src/SFA.DAS.EmployerIncentives.Database/Scripts/Generate.EmploymentCheckFailureEmailsReport.sql

```diff
@@ -139,5 +167,6 @@ INNER JOIN Accounts a ON a.AccountLegalEntityId = x.AccountLegalEntityId
 INNER JOIN IncentiveApplicationApprenticeship iaa ON iaa.Id = x.IncentiveApplicationApprenticeshipId
 INNER JOIN IncentiveApplication ia ON ia.Id = iaa.IncentiveApplicationId
 WHERE x.HasLearningRecord = 1 AND x.EmployedAt365Days = 0 AND x.PausePayments = 0
-AND ((SELECT COUNT(1) FROM incentives.EmploymentCheck ec WHERE ec.ApprenticeshipIncentiveId = x.Id AND ec.Result IS NOT NULL) = 4 -- ensure that initial and first and second 365 checks have been executed
-OR (SELECT COUNT(1) FROM archive.EmploymentCheck ec WHERE ec.ApprenticeshipIncentiveId = x.Id AND ec.Result IS NOT NULL) = 4)
+AND ((SELECT COUNT(1) FROM incentives.EmploymentCheck ec WHERE ec.ApprenticeshipIncentiveId = x.Id AND ec.Result IS NOT NULL AND CreatedDateTime < '2022-10-11') = 4 -- ensure that initial and first and second 365 checks have been executed
+OR (SELECT COUNT(1) FROM archive.EmploymentCheck ec WHERE ec.ApprenticeshipIncentiveId = x.Id AND ec.Result IS NOT NULL AND CreatedDateTime < '2022-10-11') = 4)
+AND x.ULN NOT IN (SELECT uln FROM @EmployedAtStartOfApprenticeshipFailures)
```

Comment:
@EmployedAtStartOfApprenticeshipFailures should be replaced with another temp table, populated by a query similar to the other two checks, so that the results from previous runs are excluded